app/assets/stylesheets/layout/_utilities.scss

```diff
@@ -23,3 +23,12 @@
 .bg-dark-blue {
   background: #3b5998;
 }
+
+.flex-box {
+  display: flex;
+  align-items: center;
+}
+
+.margin-top {
```

Comment:
Margin top with 30px is not really a reusable class, lets make it specific to the element you're targeting